operator/src/main/java/com/github/streamshub/console/api/v1alpha1/spec/ConsoleSpec.java

```diff
@@ -0,0 +1,55 @@
+package com.github.streamshub.console.api.v1alpha1.spec;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.sundr.builder.annotations.Buildable;
+
+@Buildable(builderPackage = "io.fabric8.kubernetes.api.builder")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ConsoleSpec {
+
+    String hostname;
```

Comment:
Are any of these fields required?

Reply:
Only `hostname` is required. It's been annotated as such in the latest commit.